tests/test_camera.py

```diff
@@ -106,7 +106,10 @@ def test_write(self):
             _, frame_out = output_video.read()
             if not ret:
                 break
-            assert np.all([frame_in, frame_out])
+            assert np.ndarray([frame_in == frame_out]).all()
```

Comment:
When I run the following snippet, it fails with the shown error:
```python
>>> frame_in = np.random.random((2,3))
>>> frame_out = frame_in
>>> np.ndarray([frame_in == frame_out])
Traceback (most recent call last):
  File "<input>", line 1, in <module>
TypeError: only integer scalar arrays can be converted to a scalar index
```

I think the proper way to write this check would be
```python
assert (frame_in == frame_out).all()
```

However, this hints at the problem that our test case apparently never reached that line. I guess this is because `input_video` reader is already released above, so it can no longer be read.

Reply:
I see, also turns out that the written video isn't identical. Visually it is but the re-encoding introduces some variance. Should we bother with the equivalence check?